config/unleash.php

```diff
@@ -5,6 +5,13 @@
   // This should be the base URL, do not include /api or anything else.
   'url' => env('UNLEASH_URL'),
 
+  // Endpoint for accessing feature flags, on the Unleash server.
+  'features_endpoint' => env('UNLEASH_FEATURES_ENDPOINT', '/api/client/features'),
```

Comment:
We're using camelCase for the keys in this file. I don't think it's the actual Laravel convention but we should be consistent in this package

Reply:
Addressed in `ffee84d85bc676ac5309c7a6ca14b5c23f0642b7`